tests/lean/hashmap_on_disk/HashmapMain/Funs.lean

```diff
@@ -0,0 +1,658 @@
+-- THIS FILE WAS AUTOMATICALLY GENERATED BY AENEAS
+-- [hashmap_main]: function definitions
+import Base.Primitives
+import HashmapMain.Types
+import HashmapMain.Opaque
+import HashmapMain.Clauses.Template
```

Comment:
We shouldn't import the `Template` file directly, but a user-written `Clauses` file (we generate the `Template` file just to help the user). I noticed this also changed for the other backends.

Reply:
I'm resolving this on the basis that your comment above says that you fixed it (and indeed the code-gen seems to indicate that this is fixed... thanks for fixing it, I think I misunderstood the intended usage, or maybe didn't realize what was going on at first)